pylibui/libui/separator.py

```diff
@@ -0,0 +1,46 @@
+"""
+ Python wrapper for libui.
+
+"""
+
+import ctypes
+from . import clibui
+
+
+class uiSeparator(ctypes.Structure):
+    """Wrapper for the uiSeparator C struct."""
+
+    pass
+
+
+def uiSeparatorPointer(obj):
+    """
+    Casts an object to uiSeparator pointer type.
+
+    :param obj: a generic object
+    :return: uiSeparator
+    """
+
+    return ctypes.cast(obj, ctypes.POINTER(uiSeparator))
+
+
+# - uiSeparator *uiNewHorizontalSeparator(void);
+def uiNewHorizontalSeparator():
+    """
+    Return a horizonal separator.
+
+    :return: HorizontalSeparator
+    """
+
```

Comment:
You should add a line that sets return type. Please check other files in this directory. You can/should use them as reference. Same for the other function. :arrow_down: